measures/ResStockArguments/measure.rb

```diff
@@ -243,6 +273,22 @@ def run(model, runner, user_arguments)
       args['geometry_has_flue_or_chimney'] = 'true'
     end
 
+    if args['cooling_system_rated_cfm_per_ton'].is_initialized && args['cooling_system_actual_cfm_per_ton'].is_initialized
+      args['cooling_system_airflow_defect_ratio'] = (args['cooling_system_actual_cfm_per_ton'].get / args['cooling_system_rated_cfm_per_ton'].get) / args['cooling_system_rated_cfm_per_ton'].get
```

Comment:
Shouldn't these be `(actual - rated)/rated` instead of `(actual / rated) / rated`?